### PR TITLE
Define security contacts on a number of plugins

### PR DESCRIPTION
**Pull request by batmat**
# Description

@Vlatombe @jtnord @jglick declared currently as developer on plugins of that list;

Approved by an existing maintainer:
* [x] https://github.com/jenkinsci/aws-credentials-plugin
* [x] https://github.com/jenkinsci/azure-publishersettings-credentials-plugin
* [x] https://github.com/jenkinsci/kubernetes-client-api-plugin
* [x] https://github.com/jenkinsci/mapdb-api-plugin
* [x] https://github.com/jenkinsci/mesos-plugin

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by batmat** [comment]
I checked `azure-publishersettings-credentials-plugin` as a current developer declared for it.

**Review by Vlatombe — APPROVED** [review]

**Review by jglick — APPROVED** [review]

File: permissions/plugin-mesos.yml [hunk 1]
```diff
@@ -12,3 +12,6 @@ developers:
 - "maselvaraj"
 - "vinodkone"
 - "vlatombe"
+security:
+  contacts:
+    jira: "modern_platforms_security_members"
```
Comment (jtnord) [inline permissions/plugin-mesos.yml:17]: not maintained by cloudbees?
```suggestion
```

**Comment by timja** [comment]
The repo for the azure plugin is empty?

**Comment by batmat** [comment]
True Tim. This plugin is actually "tombstoned" indeed. No code anymore and it then got a last release to make sure users cannot have issues with it anymore.
Rationale is https://github.com/jenkinsci/azure-publishersettings-credentials-plugin/commit/634cfb6d8ad223c38b32fb326e4b46921ad17568 AFAICT.

File: permissions/plugin-mapdb-api.yml [hunk 2]
```diff
@@ -5,4 +5,7 @@ issues:
 - jira: '18829' # mapdb-api-plugin
 paths:
 - "org/jenkins-ci/plugins/mapdb-api"
-developers: []
\ No newline at end of file
+developers: []
```
Comment (stephenc) [inline permissions/plugin-mapdb-api.yml:8]: 👍

**Review by stephenc — APPROVED** [review]

File: permissions/plugin-mapdb-api.yml (hunk 2 above)
Comment (batmat) [reply permissions/plugin-mapdb-api.yml:8]: Thanks Stephen!

**Comment by batmat** [comment]
Now we got Stephen's approval, we can merge.

**Pull request merged**